frontend/src/jobRoles.js

```diff
@@ -0,0 +1,15 @@
+const fetch = require('node-fetch')
+const api_url = process.env.API_URL;
+
+exports.getJobRoles = async () => {
+    const response = await fetch(`${api_url}/job-roles`,{method:'GET',headers:{}})
+    return await response.json();
+};
+
+exports.getJobRole = async (id) => {
+    const response = await fetch(`${api_url}/job-roles/${id}`,{method:'GET',headers:{}})
+    const roleData = await response.json();
+    const bandResponse = await fetch(`${api_url}/bands/${roleData.bandID}`,{method:'GET',headers:{}})
+    const bandData = await bandResponse.json();
+    return [roleData, bandData];
```

Comment:
I suggest making the returned value an object, so that you can refer to each part with a meaningful name instead of numeric indices.
```suggestion
    return { roleData, bandData };
```